### PR TITLE
solution 1

### PR DESCRIPTION
**Pull request by kostya-kosyuk**
[DEMO LINK](https://kostya-kosyuk.github.io/react_decompose/)

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,6 +1,15 @@
 // import the React library here
+import React from 'react';
+import Navigation from '../Navigation/Navigation';
```
Comment (andrii-dmytruk) [inline src/components/Header/Header.jsx:3]: fix path

**Review by andrii-dmytruk — APPROVED** [review]
GJ! But fix according to the comment

**Review by AndriiRut — APPROVED** [review]
good job 🚀

**Comment by kostya-kosyuk** [comment]
> good job 🚀

**Pull request closed without merging**